runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1439,6 +1439,10 @@ gcParseXXArguments(J9JavaVM *vm)
 				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_VALUE_MUST_BE_ABOVE, "-XX:CheckpointGCThreads=", (UDATA)0);
 				goto _error;
 			}
+			if (extensions->checkpointGCthreadCount > extensions->gcThreadCount) {
+				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_GC_OPTIONS_MUST_BE_NO_GREATER_THAN, "-XX:CheckpointGCThreads=", VMOPT_XGCTHREADS);
+				goto _error;
```

Comment:
The JVM shouldn't fail, but a warning message should be printed indicating that the option wasn't respected.